segmentation/utils/synthetic_spatial_datagen.py

```diff
@@ -7,35 +7,38 @@
 import xarray as xr
 import copy
 
+from random import seed
+from random import randint
 from segmentation.utils import spatial_analysis_utils as sau
+from segmentation.utils import visualize as viz
 from scipy.spatial.distance import cdist
 from skimage.measure import label
 
 
-def direct_init_dist_matrix(num_A=100, num_B=100, num_C=100,
-                            distr_AB={'mean': 100, 'var': 1}, distr_AC={'mean': 20, 'var': 1},
-                            seed=None):
+def generate_test_dist_matrix(num_A=100, num_B=100, num_C=100,
+                              distr_AB=(10, 1), distr_random=(200, 1),
+                              seed=None):
     """
-    This function will return a random dist matrix such that the distance between cells
-    of types A and B are overall larger than the distance between cells of types A and C
+    This function will return a random dist matrix specifying the distance between cells of
+    types A and B and between cells of all other groups (type C).
 
     Each row and column representing a cell.
     We generate the points using Gaussian distributions
     Ideally, the parameters for A to B distances will be set such that they produce a lower range of values
     than A to C distances.
 
-    Will return a random distance matrix as well as the dictionary of associated cell: label IDs
-    The above is generated from the generate_labels function
+    Note that these distance matrices created are non-Euclidean.
```

Comment:
😆

Reply:
( ▀ ͜͞ʖ▀)

we have now entered big nerd hours